README.md

```diff
@@ -291,6 +284,71 @@ One other tip... This totally works:
 </glamorous.Div>
 ```
 
+#### Overriding component styles
+
+The most common scenario for using props is to override the style of an
+existing component (generated by `glamorous` or not). In this section we'll
+explain how you can use `className`, `css` and composition to override the
+styles of a component. Let's see how that can be done in the examples below.
+
+> Try this out in your browser [here](https://codesandbox.io/s/Kro5369wG)!
+
+We'll use this as our `GlamorousComponent`:
+```javascript
```

Comment:
This one's fine to be `javascript` because there's no `jsx`. Could you put a blank line above this one though? I like to give all my code fences some space.